lighthouse-core/lib/network-recorder.js

```diff
@@ -16,12 +17,15 @@ const IGNORED_NETWORK_SCHEMES = ['data', 'ws'];
 class NetworkRecorder extends EventEmitter {
   /**
    * Creates an instance of NetworkRecorder.
-   * @param {Array<LH.WebInspector.NetworkRequest>} recordArray
    */
-  constructor(recordArray) {
+  constructor() {
     super();
 
-    this._records = recordArray;
+    /** @type {NetworkRequest[]} */
+    this._records = [];
+    /** @type {Map<string, NetworkRequest>} */
+    this._recordsById = new Map();
+
     this.networkManager = NetworkManager.createWithFakeTarget();
```

Comment:
do we still need this around? followup?

Reply:
that one's totally safe to go right now actually 👍